packages/core/content-type-builder/admin/src/components/ContentTypeBuilderNav/useContentTypeBuilderMenu.js

```diff
@@ -111,12 +96,36 @@ const useContentTypeBuilderMenu = () => {
     toggleNotification({
       type: 'info',
       message: {
-        id: `${getTrad('notification.info.creating.notSaved')}`,
+        id: getTrad('notification.info.creating.notSaved'),
         defaultMessage: 'Please save your work before creating a new collection type or component',
       },
     });
   };
 
+  const componentsData = Object.entries(componentsGroupedByCategory)
```

Comment:
Should these be memoized too?

Reply:
They weren't before and in the interest of preservation i wanna keep it as close to how it was prior 😛